include/gps.hpp

```diff
@@ -0,0 +1,58 @@
+#ifndef OBC_GPS_HPP
+#define OBC_GPS_HPP
+
+#include <Adafruit_GPS.h>
+
+#include <array>
+
+#include "error.hpp"
+
+namespace obc {
+
+constexpr bool GPSECHO = false;
+constexpr std::array<int, 3> format_helper = {10, 9, 100};
+constexpr double velocity_conversion = 1.85166;
+
+using Gps = Adafruit_GPS;
+using Gps_float = nmea_float_t;
+
+struct GpsTime {
+    uint8_t hour;
+    uint8_t minute;
+    uint8_t seconds;
+    uint16_t milliseconds;
+};
+
+struct GpsDate {
+    uint8_t year;
+    uint8_t month;
+    uint8_t day;
+};
+
+struct GpsPosition {
+    bool fix;
+    uint8_t fixquality;
+    Gps_float longitude;
+    char lon;
+    Gps_float latitude;
+    char lat;
+    Gps_float altitude;
+    Gps_float speed;
+    uint8_t satelites;
+};
+
+struct GpsMeasurments {
```

Comment:
Tutaj sugerując się ostatnią uwagą @wiktorwieclaw ta struktura powinna nazywać się GpsMeasurementsResult.

Reply:
Jak wsadzimy do jednego structa pomiary to się rozwiąże sprawa też z printem

Reply:
To pójdę w wariant z jedną funkcją, tylko boli mnie trochę, że będzie taka dodatkowa pusta struktura. W tej głównej funkcji printującej oddzielić jakoś (komentarzem np) gdzie co jest wypisywane?